test/e2e/onFCP-test.js

```diff
@@ -235,6 +235,23 @@ describe('onFCP()', async function() {
     assert.strictEqual(fcp2.navigationType, 'back-forward-cache');
   });
 
+  it('reports restore as nav type for wasDiscarded', async function() {
```

Comment:
This looks good to me, but it also makes me realize that we don't have any tests for prerendering. We don't necessarily need to do it as part of this PR, but eventually I think it'd be good to update this test (in other test files as well) to just check all the different `navigationTypes` in a single test. WDYT?

Reply:
We do for prerender:

https://github.com/GoogleChrome/web-vitals/blob/43a116b73f6bebcb409b336564eba93a9a4525a2/test/e2e/onFCP-test.js#L55-L76

Reply:
Yeah, though I guess only only some of the tests. I'd originally search `onCLS-test.js` and didn't see it.

I think it's fine to merge this PR as is. We can address this at some point in the future given that we do have some tests for it now.

Reply:
Yeah looks like we're missing for CLS, INP, and FID. Should be easy enough to add so gimme 5 mins, and if trickier than that then will leave until later.